test/data/search/8bins20window0error50pattern40overlap.out

```diff
@@ -0,0 +1,32 @@
+0	0,
+1	0,
+2	0,
+3	0,
+4	1,
+5	1,
+6	1,
+7	1,
+8	2,
+9	2,
+10	2,
+11	2,
+12	3,
+13	3,
+14	3,
+15	3,
+16	4,
+17	4,
+18	4,
+19	4,
```

Comment:
Here all bins are found correctly, but if window size is 23 i.e the probabilistic thresholding is used instead then no matches are found.